tools/subgraph-example/hardhat.config.ts

```diff
@@ -19,7 +22,7 @@ task("deployERC20", "Deploys ERC20 contract", async (taskArgs, hre) => {
     contractName,
     wallet
   );
-  const contract = await ERC20.deploy(20);
+  const contract = await ERC20.deploy(20, {gasLimit: 500_000});
```

Comment:
Q: what makes it so that now we have to specific the gasLimit?
What changed is it it that the gasLimit returned by the relay currently isn't sufficient?

Reply:
Something with the latest version of the relay. It returns 21k gas, but in fact this operation needs more. We have to address this in the relay logic itself.

Reply:
is it the `relay` ? I've encountered similar issues but is with the `mirror-node` of specific versions of the `hedera-local-node`.

I have been more successful with `1.10.0` version of hedera-local-node

Reply:
This may be related to HIP 514 flows.
If thr mirror node logic is returning 21k then it's a bug there.
If the relay is dealing with a mirror node 400 and then failing back to return a 21k gas then we have a regression.

We should double check this as it seems to impact this and other tests.